demo/index.html

```diff
@@ -29,13 +29,13 @@
   }
   </style>
 </head>
-<body layout vertical center-justified center unresolved>
+<body layout vertical center-justified center>
```

Comment:
it appears to be back, at least testing today in 1.0.3. But my comment was also to move the layout attributes to classes.